gyp_xwalk

```diff
@@ -351,6 +351,10 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  if gyp_vars_dict.get('OS') == 'win':
+    args.append('-Denable_widevine=1')
+    args.append('-Denable_pepper_cdms=1')
```

Comment:
This setting doesn't work is because <code>gyp_vars_dict.get('OS')</code> is <code>None</code>. Here use <code>sys.platform == 'win32'</code> to instead.

>BTW, isn't -Denable_widevine=1 -Denable_pepper_cdms=1 need for android build?

Android don't need these flags. For android, the platform has integrated widevinecdm. It don't use this path.

Another thing is I removed <code>-Denable_pepper_cdms=1</code> since the default value is 1 for windows. So only <code> -Denable_widevine=1</code> is needed.